src/odemis/acq/stream/_base.py

```diff
@@ -561,6 +636,65 @@ def _getDetectorVA(self, vaname):
                 raise AttributeError(u"Detector has not VA %s" % (vaname,))
             return hwva
 
+    def _linkHwAxes(self):
```

Comment:
BTW, there are 2 streams which already have this method. You need to check that they don't break, now they are overloading it (might be just a matter of calling also the `super()`).
```
> grep -Ir linkHwAxes src/odemis/ plugins/
plugins/la_spec.py:            # self._linkHwAxes()
plugins/la_spec.py:            self._unlinkHwAxes()
plugins/la_spec.py:        self._linkHwAxes()
plugins/la_spec.py:    def _linkHwAxes(self, _=None):
plugins/la_spec.py:    def _unlinkHwAxes(self):
plugins/la_spec.py:        self._sccd._linkHwAxes()
plugins/la_spec.py:            self._sccd._unlinkHwAxes()
src/odemis/acq/stream/_helper.py:            self._linkHwAxes()
src/odemis/acq/stream/_helper.py:            self._unlinkHwAxes()
src/odemis/acq/stream/_helper.py:    def _linkHwAxes(self):
src/odemis/acq/stream/_helper.py:    def _unlinkHwAxes(self):
```

Reply:
Seems like the fix is simple for this. Added it.